package/Codegen/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Codegen": {
       "commandName": "Project",
-      "commandLineArgs": "-verbose -winmd ..\\..\\..\\..\\..\\example\\windows\\packages\\Microsoft.UI.Xaml.2.6.0\\lib\\uap10.0\\Microsoft.UI.Xaml.winmd"
+      "commandLineArgs": "-verbose -winmd ..\\..\\..\\..\\..\\examplenuget\\windows\\packages\\Microsoft.UI.Xaml.2.6.0\\lib\\uap10.0\\Microsoft.UI.Xaml.winmd"

```

Comment:
Feel free to remove this change. For me, the winmd reference from example was not working so used this.